easybuild/easyconfigs/u/UCC/UCC-1.2.0-GCCcore-13.2.0.eb

```
@@ -20,6 +20,11 @@ checksums = [
     {'UCC-1.1.0-multiple_component_paths.patch': '3081d0f694331daa4a88a0fa3fb54b9a918015248ae5eb7b3157b924abd31bee'},
 ]
 
+if ARCH == "riscv64":
+    patches += ['UCC-1.2.0_add-riscv-support.patch']
+    checksums += [{'UCC-1.2.0_add-riscv-support.patch': 
```

Sorry, I forgot to remove the trailing whitespace
```suggestion
    checksums += [{'UCC-1.2.0_add-riscv-support.patch':
```